packages/contracts/scripts/shared/types/forge.ts

```diff
@@ -0,0 +1,9 @@
+export type ForgeArguments = {
+    name: string,
+    network: string,
+    rpcUrl: string,
+    privateKey: string,
+    contractInstance: string,
+    constructorArguments: string[],
+    etherscanApiKey?: string,
```

Comment:
Shouldn't this be in an `.env` file?

Reply:
They're in the `.env` file. but to handle some missing cases of those values and load default values for local chain, I had them defined as `ForgeArguments`.